### DISCUSSION TITLE
How to add to the site robots.txt?

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#9702] [Q&A]
I did not find anything on this topic in the documentation(https://docs.orchardcore.net/en/dev/).
I found issue(https://github.com/OrchardCMS/OrchardCore/issues/3219) where it seems to be already implemented, you can tell me how to enable it correctly? 
Thanks in advance.

**Comment by hishamco** [comment]
I think @deanmarcussen was on charge of doing this module if I'm not wrong

**Comment by deanmarcussen** [comment]
We never did a robots.txt editor into Orchard Core (just sitemaps)

I have one here you are welcome to use https://github.com/ThisNetWorks/ThisNetWorks.OrchardCore.Robots

**Reply by Skrypt** [reply]
This is a pretty standard module that everyone will need to use at some point. Maybe worth doing a demo and proposing adding this to OC by default. Else, we need to list those contrib modules somewhere.

**Comment by ns8482e** [comment]
For single tenant you can create static file `robots.txt` in `wwwroot` folder.

If you need for multi-tenant, There is a feature called `Static File Provider` once you enable that feature - you can define static files per tenant within `App_Data\Sites\[tenantname]\wwwroot` folder and you can have robots.txt for each tenant .
e.g `App_Data\Sites\Tenant1\wwwroot\robots.txt`, `App_Data\Sites\Tenant2\wwwroot\robots.txt`

**Reply by Skrypt** [reply]
@ns8482e Could you document this in OC doc if it's not already there?

**Reply by Skrypt** [reply]
https://docs.orchardcore.net/en/latest/docs/reference/modules/Tenants/index.html#static-file-provider-feature
